pom.xml

```diff
@@ -212,6 +212,11 @@
     <dependencies>
         <!-- This explicit dependency is added because the older version
             which gets added by parent pom has the vulnerability -->
+        <dependency>
+            <groupId>org.jenkins-ci.modules</groupId>
+            <artifactId>instance-identity</artifactId>
+            <version>116.vf8f487400980</version>
```

Comment:
Let's use <properties> for the version rather than adding the version here

Reply:
Instance identity is not available if it's not specified as a dependency